src/Neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -36,6 +36,11 @@ partial class ApplicationEngine
         /// </summary>
         public const int MaxNotificationSize = 1024;
 
+        /// <summary>
+        /// The maximum number of notifications per application execution.
+        /// </summary>
+        public const int MaxNotificationCount = 512;
```

Comment:
Use `ExecutionEngineLimits` Class `engine.Limits.MaxNotificationCount`

Reply:
Notifications is a syscall, outside from VM

Reply:
Its still an `Engine` Limitation, just cause the VM is splitted into two libraries shouldn't make it be to separate things.

Reply:
We have `MaxNotificationSize` and `MaxEventName` declared in the same class as `MaxNotificationCount`, these three constants must be kept together.